CHANGELOG

```diff
@@ -111,6 +111,7 @@ Other:
 - Added function utils.auth.anonymous_is_enabled which returns true, if it is.
 - Changed has_perm to support an user id or None (for anyonmous) as first argument.
 - Removed our AnonymousUser. Make sure not to use user.has_perm() anymore.
+- Add the command getgeiss to download the latest version of geiss.
```

Comment:
You told me, that you only need this lines as an hint, and that you rewrite the file anyway.